デスクトップアプリ「AtCoder Review 」が手元でコンパイルできない気がする

## Issue

- 表題のアプリがビルドできない
- 作者が公開されているappも動作が確認できていない状況

## Env

+ macOS Mojave version 10.14.6
+ VS Code 1.51.1
+ Docker 2.3.0.5

## 試したこと

+ VS CodeとDockerを利用してビルドを試みた
  + ビルドそのものはできるが、作成したappの初期設定でAtCoderのユーザデータの取得に失敗する

+ ローカル環境にNode.jsとyarnを用意
  + yarn dist:macを実行すると、途中でビルドが失敗する
  + エラーメッセージに記載されているパッケージを追加でインストール
  + Node.jsのVersionを12.19.1と14.15.1の2種類試した
  + 以下のエラーメッセージに対して、該当するパッケージを追加でインストールしたが解決できず
 
```terminal
Hash: cc0f4eb6b5a346a6a098c9f61f48bc25c50bb8f7
Version: webpack 4.42.1
Child
    Hash: cc0f4eb6b5a346a6a098
    Time: 463ms
    Built at: 2020-11-23 16:06:03
     2 assets
    Entrypoint main = main.js main.js.map
    [0] multi ./src/main/index.ts 28 bytes {0} [built]
    
    ERROR in multi ./src/main/index.ts
    Module not found: Error: Can't resolve 'ts-loader' in '/xxx/atcoder-review'
     @ multi ./src/main/index.ts main[0]
Child
    Hash: c9f61f48bc25c50bb8f7
    Time: 377ms
    Built at: 2020-11-23 16:06:03
     2 assets
    Entrypoint renderer = renderer.js renderer.js.map
    [0] multi ./src/renderer/index.tsx 28 bytes {0} [built]
    
    ERROR in multi ./src/renderer/index.tsx
    Module not found: Error: Can't resolve 'ts-loader' in '/xxx/atcoder-review'
     @ multi ./src/renderer/index.tsx renderer[0]
    Child HtmlWebpackCompiler:
        
        ERROR in Entry module not found: Error: Can't resolve 'html-loader' in '/xxx/atcoder-review'
```

## 考えられる原因

- Node.js、yarn、パッケージのバージョンの関係で、破壊的な変更が行われた影響?
- TypeScriptに関する設定ファイルで、必要な設定が記載されていない可能性がある
  - 自分が導入したバージョンだと追加設定が必要で、作者の方の環境だと不要みたいな可能性はある? 

## 対策

- 上記のエラーに対して、GitHubレポジトリのIssueを調べる
- バージョンをいくつか試してみる
- 作者に問い合わせる

--- a/archived.md
+++ b/archived.md
@@ -65,6 +65,14 @@ filename: archived
     <img loading = "lazy" src="images/chrome_extension/coding_shout.png" alt="coding shout">
   </div>
 
+- [AtCoder Review](https://github.com/jueve/atcoder-review) - コンテストの復習に焦点を置いたクロスプラットフォームのデスクトップアプリケーション。
+  - 理由: ドキュメントの記載はないが、ソースコードのビルドおよび配布されているインストーラの動作保証が困難な状況のため。
+  - 代替手段: 調査中。
+
+  <div align="center">
+    <img loading = "lazy" src="images/web_app/atcoder_review.png" alt="atcoder review">
+  </div>
+
 ## リンク切れと思われるサービス、ツールなど
 
 リンクが切れていると思われるサービス・ツールなどを掲載しています。一定の期間(3〜6か月程度)更新されない場合は、下記の「開発、提供が終了したサービス、ツールなど」へ移動させたいと思います。

--- a/web_app.md
+++ b/web_app.md
@@ -372,14 +372,6 @@ filename: web_app
 
 ## その他
 
-### Desktop App
-
-- [AtCoder Review](https://github.com/jueve/atcoder-review) - コンテストの復習に焦点を置いたクロスプラットフォームのデスクトップアプリケーション。
-
-  <div align="center">
-    <img loading = "lazy" src="images/web_app/atcoder_review.png" alt="atcoder review">
-  </div>
-
 ### Google Apps Script
 
 - [AtCoder Daily AC Checker](https://github.com/purple-jwl/atcoder-daily-ac-checker) - AC(Accepted)している人をSlack上で褒める（紹介する）。

--- a/web_app.md
+++ b/web_app.md
@@ -22,7 +22,6 @@ filename: web_app
   - [Twitter](#twitter)
 - [Web API](#web-api)
 - [その他](#%E3%81%9D%E3%81%AE%E4%BB%96)
-  - [Desktop App](#desktop-app)
   - [Google Apps Script](#google-apps-script)
   - [Google Sheets](#google-sheets)
   - [IDE](#ide)

